roles/rsyslog/roles/input_roles/ovirt/defaults/main.yaml

```diff
@@ -90,6 +90,11 @@ rsyslog_conf_ovirt_formatting:
               action(type="mmjsonparse" cookie="") # parse entire message as json
               set $!@timestamp = exec_template('cnvt_to_viaq_timestamp');
               unset $!time;
+              if $!collectd!plugin == 'virt' then {
```

Comment:
@nhosoi is this safe to do?  Or should there first be a check for `strlen($!collectd!plugin) > 0`

Reply:
@nhosoi @richm  In Collectd the  $!collectd!plugin always exists.

Reply:
ok